### PR TITLE
removed xml

### PR DESCRIPTION
**Pull request by Juan-EBI**
- removed xml task

### CODE REVIEW

File: Exporter/build.gradle [hunk 1]
```diff
@@ -55,8 +55,8 @@ dependencies {
     runtime("com.h2database:h2:1.4.196")
 
     testCompile("org.springframework.boot:spring-boot-starter-test")
-    testCompile("com.jamesmurty.utils:java-xmlbuilder:1.2")
     testCompile("org.mockftpserver:MockFtpServer:2.0.2")
     testCompile("org.json:json:20171018")
+    testCompile("com.jamesmurty.utils:java-xmlbuilder:1.2")
```
Comment (jhoanmanuelms) [inline Exporter/build.gradle]: Do we still need this?

File: Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/job/SubmissionExporter.java [hunk 2]
```diff
@@ -18,11 +18,11 @@
 import uk.ac.ebi.biostd.exporter.jobs.full.configuration.FullExportFileProperties;
 import uk.ac.ebi.biostd.exporter.jobs.full.configuration.FullExportJobProperties;
 import uk.ac.ebi.biostd.exporter.jobs.full.json.JsonBufferedFileWriter;
-import uk.ac.ebi.biostd.exporter.jobs.full.xml.XmlBufferedFileWriter;
 
 @Component
 @AllArgsConstructor
 public class SubmissionExporter {
+
     public static final String XML_EXTENSION = ".xml";
```
Comment (jhoanmanuelms) [inline Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/job/SubmissionExporter.java]: Same here

File: Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/job/SubmissionExporter.java [hunk 3]
```diff
@@ -46,7 +46,7 @@ public QueueJob getJoinJob(
                 .reader(new BlockingQueueRecordReader(processQueue, workers))
                 .filter(recordFilter)
                 .processor(recordProcessor)
-                .writer(getRecordWriter(fileName, fileExtension))
+                .writer(getRecordWriter(fileName))
```
Comment (jhoanmanuelms) [inline Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/job/SubmissionExporter.java]: Couldn't it just be a JsonBufferedFileWriter directly?

File: Exporter/src/test/java/uk/ac/ebi/biostd/exporter/FullExportTest.java [hunk 4]
```diff
@@ -47,15 +42,14 @@
...
         "classpath:scripts/sql/public_submission.sql"})
-@Sql(executionPhase = AFTER_TEST_METHOD, scripts = { "classpath:scripts/sql/drop_schema.sql" })
+@Sql(executionPhase = AFTER_TEST_METHOD, scripts = {"classpath:scripts/sql/drop_schema.sql"})
 public class FullExportTest extends BaseIntegrationTest {
+
     private static final String FULL_EXPORT_NAME = "studies";
     private static final String PUBLIC_ONLY_EXPORT_NAME = "publicOnlyStudies";
     private static final String NOTIFICATION_URL = "http://localhost:8181/api/update/full";
     private static final String EXPECTED_OUTPUT_PATH = "/test_files/";
-    private static final String EXPECTED_FULL_XML_PATH = EXPECTED_OUTPUT_PATH + "full.xml";
```
Comment (jhoanmanuelms) [inline Exporter/src/test/java/uk/ac/ebi/biostd/exporter/FullExportTest.java]: I think we also should remove the corresponding files

File: Exporter/build.gradle (hunk 1 above)
Comment (Juan-EBI) [reply Exporter/build.gradle]: yes it use for unit testing xml generated files.

**Pull request merged**